PhillipsAuctionHouse/seldon
# feat(readmore): L3-3579 Read More and Collapsible Component

**Jira ticket**

[L3-3579](https://phillipsauctions.atlassian.net/browse/L3-3579)

**Screenshots**
Read More
| Close | Open |
| ------ | ----- |
|<img width="820" alt="Screenshot 2024-09-06 at 4 16 47 PM" src="https://github.com/user-attachments/assets/173c8594-48df-45c3-8f1c-c59e3baa31b8">|<img width="798" alt="Screenshot 2024-09-06 at 4 17 25 PM" src="https://github.com/user-attachments/assets/1b270cbc-eaca-47f0-9478-842eaeb76301"> |

Collapsible
| Close | Open |
| ------ | ----- |
|<img width="714" alt="Screenshot 2024-09-06 at 4 24 59 PM" src="https://github.com/user-attachments/assets/aca84b2c-5e20-440f-b8df-7e45c94d64f1">|<img width="798" alt="Screenshot 2024-09-06 at 4 24 50 PM" src="https://github.com/user-attachments/assets/e5284377-628d-4af0-b0cb-7adc18bc1966">|

**Summary**

Added a Read More component and a Collapsible Component

Collapsible is just a port of the [Radix UI Collapsible](https://www.radix-ui.com/primitives/docs/components/collapsible)

Read more consumes the port & the seldon Button component

**Change List (describe the changes made to the files)**

- Additions, Changes, and Removals
- New Collapsible and ReadMore components

**Acceptance Test (how to verify the PR)**

- Add step by step instructions on how to verify the change
- Pull down branch
- Run storybook
- Check out ReadMore & Collapsible components

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc



**Things to look for during review**


New Components



[L3-3579]: https://phillipsauctions.atlassian.net/browse/L3-3579?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
